packages/react-hooks/src/inbox/reducer.ts

```diff
@@ -62,24 +61,35 @@ import {
   SetCurrentTab,
   INBOX_SET_CURRENT_TAB,
 } from "./actions/set-current-tab";
+import { IInboxMessagePreview } from "@trycourier/client-graphql";
 
 export const mapMessage = (
-  message: IGraphMessageResponse,
+  message: IInboxMessagePreview,
   lastMarkedAllRead?: number
 ): IMessage => ({
-  blocks: message.content.blocks,
-  body: message.content.body,
+  blocks: [
+    {
+      type: "text",
+      text: message.preview,
+    },
+    ...(message?.actions ?? []).map((a) => ({
+      type: "action",
+      text: a.content,
+      url: a.href,
+    })),
+  ],
+  body: message.preview,
   created: message.created,
-  data: message.content.data,
+  data: message.data,
   messageId: message.messageId,
   opened: message.opened,
   read: lastMarkedAllRead
     ? lastMarkedAllRead > new Date(message.created).getTime()
       ? true
       : message.read
     : message.read,
-  title: message.content.title,
-  trackingIds: message.content.trackingIds,
+  title: message.title,
+  //trackingIds: message.content.trackingIds,
```

Comment:
need this?

Reply:
im not sure exactly tbh, the new inbox apis have all the tracking ids in the db so we don't need to pass them back